app/models/match_decisions/confirm_shelter_agency_decline_dnd_staff.rb

```diff
@@ -81,9 +81,9 @@ def decline_confirmed
         match.rejected!
       end
 
-      def canceled
+      def canceled contact
         Notifications::MatchCanceled.create_for_match! match
-        match.canceled!
+        match.canceled! contact&.id
```

Comment:
`ClientOpportunityMatch#canceled!` takes a named parameter, so this won't work as written.